doc/make.py

```diff
@@ -1,232 +0,0 @@
-#!/usr/bin/env python
-"""Wrapper script for calling Sphinx. """
-
-from __future__ import print_function
-import glob
-import os
-import shutil
-import sys
-import re
-import argparse
-import subprocess
-import matplotlib
-import six
-
-
-def copy_if_out_of_date(original, derived):
-    """Copy file only if newer as target or if target does not exist. """
-    if (not os.path.exists(derived) or
-        os.stat(derived).st_mtime < os.stat(original).st_mtime):
-        try:
-            shutil.copyfile(original, derived)
-        except IOError:
-            if os.path.basename(original) == 'matplotlibrc':
-                msg = "'%s' not found. " % original + \
-                      "Did you run `python setup.py build`?"
-                raise IOError(msg)
-            else:
-                raise
-
-
-def check_build():
-    """Create target build directories if necessary. """
-    build_dirs = ['build', 'build/doctrees', 'build/html', 'build/latex',
-                  'build/texinfo', '_static', '_templates']
-    for d in build_dirs:
-        try:
-            os.mkdir(d)
-        except OSError:
-            pass
-
-
-def doctest():
-    """Execute Sphinx 'doctest' target. """
-    subprocess.call(
-        [sys.executable]
-        + '-msphinx -b doctest -d build/doctrees . build/doctest'.split())
-
-
-def linkcheck():
-    """Execute Sphinx 'linkcheck' target. """
-    subprocess.call(
-        [sys.executable]
-        + '-msphinx -b linkcheck -d build/doctrees . build/linkcheck'.split())
-
-DEPSY_PATH = "_static/depsy_badge.svg"
-DEPSY_URL = "http://depsy.org/api/package/pypi/matplotlib/badge.svg"
-DEPSY_DEFAULT = "_static/depsy_badge_default.svg"
-
-
-def fetch_depsy_badge():
```

Comment:
Can this be ported over as well?  This works around depsy not having an https end point which results in matplotlib.org warning about non-secure resources.

Reply:
I am also ok with just removing the depsy badge, it seems to not have been updated recently.

Reply:
I'll deal with that once I figure out the examples not running issue.

Reply:
I'm in favor of removing all the badges that we have. We have a problem of signal to noise ratio on the website, so we need to think about what is necessary and what can be remove, and I think this can safely been remove (basically, those badges are only understood by people who already know this information).